src/main/java/com/seam/api/SeamBuilder.java

```diff
@@ -19,6 +20,11 @@ public final class SeamBuilder {
 
     private Environment environment = Environment.DEFAULT;
 
+    public SeamBuilder apiKey(String apiKey) {
+        this.clientOptionsBuilder.addHeader("Authorization", "Bearer " + apiKey);
+        return this;
+    }
```

Comment:
@kainpets why did you add this?

Reply:
The testUtils fails without it at line 45

Reply:
It's copied from the previous version